media/transcoder/create_job_with_concatenated_inputs.py

```diff
@@ -0,0 +1,202 @@
+          (e.g., '3.5s'), relative to the second input video timeline.
+        end_time_input2: End time, in fractional seconds ending in 's'
+          (e.g., '15s'), relative to the second input video timeline.
+        output_uri: Uri of the video output folder in the Cloud Storage bucket."""
+
+    s1 = duration.Duration()
+    s1.FromJsonString(start_time_input1)
+    e1 = duration.Duration()
+    e1.FromJsonString(end_time_input1)
+
+    s2 = duration.Duration()
+    s2.FromJsonString(start_time_input2)
+    e2 = duration.Duration()
+    e2.FromJsonString(end_time_input2)
+
+    client = TranscoderServiceClient()
+
+    parent = f"projects/{project_id}/locations/{location}"
+    job = transcoder_v1.types.Job()
+    job.output_uri = output_uri
+    job.config = transcoder_v1.types.JobConfig(
+        inputs=[
+            transcoder_v1.types.Input(
+                key="input1",
+                uri=input1_uri,
+            ),
+            transcoder_v1.types.Input(
+                key="input2",
+                uri=input2_uri,
+            ),
+        ],
+        edit_list=[
+            transcoder_v1.types.EditAtom(
+                key="atom1",
+                inputs=["input1"],
+                start_time_offset=s1,
+                end_time_offset=e1,
+            ),
+            transcoder_v1.types.EditAtom(
+                key="atom2",
+                inputs=["input2"],
+                start_time_offset=s2,
+                end_time_offset=e2,
+            ),
+        ],
+        elementary_streams=[
+            transcoder_v1.types.ElementaryStream(
+                key="video-stream0",
+                video_stream=transcoder_v1.types.VideoStream(
+                    h264=transcoder_v1.types.VideoStream.H264CodecSettings(
+                        height_pixels=360,
+                        width_pixels=640,
+                        bitrate_bps=550000,
+                        frame_rate=60,
+                    ),
+                ),
+            ),
+            transcoder_v1.types.ElementaryStream(
+                key="audio-stream0",
+                audio_stream=transcoder_v1.types.AudioStream(
+                    codec="aac", bitrate_bps=64000
+                ),
+            ),
+        ],
+        mux_streams=[
+            transcoder_v1.types.MuxStream(
+                key="sd",
+                container="mp4",
+                elementary_streams=["video-stream0", "audio-stream0"],
+            ),
+        ],
+    )
+    response = client.create_job(parent=parent, job=job)
+    print(f"Job: {response.name}")
+    return response
+
+
+# [END transcoder_create_job_with_concatenated_inputs]
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--project_id", help="Your Cloud project ID.", required=True)
+    parser.add_argument(
+        "--location",
+        help="The location to start this job in.",
```

Comment:
Obligatory note that we don't require you to write CLIs for samples. (See [note](https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/snippets.md?cl=head#running-from-the-command-line) in the samples style guide). 

You are welcome to do so if you think it's useful for your product or to keep newer samples consistent with older ones.

Reply:
Thanks. I will keep these for consistency.